Pilot/pilotTools.py

```diff
@@ -19,6 +19,15 @@
 from functools import partial, wraps
 from threading import RLock
 
+try:
+    from Pilot.proxyTools import (
+        X509BasedRequest
+    )
+except ImportError:
+    from proxyTools import (
+        X509BasedRequest
+    )
+
```

Comment:
This can be added below, where there is 
```
try:
    from Pilot.proxyTools import getVO
except ImportError:
    from proxyTools import getVO
```

simply doing 

```python

try:
    from Pilot.proxyTools import getVO, X509BasedRequest
except ImportError:
    from proxyTools import getVO, X509BasedRequest
```